pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -179,17 +191,41 @@ func (pas *PodAutoscalerStatus) MarkInactive(reason, message string) {
 // MarkResourceNotOwned changes the "Active" condition to false to reflect that the
 // resource of the given kind and name has already been created, and we do not own it.
 func (pas *PodAutoscalerStatus) MarkResourceNotOwned(kind, name string) {
+	// TODO: This looks more like a bootstrap condition?
 	pas.MarkInactive("NotOwned",
 		fmt.Sprintf("There is an existing %s %q that we do not own.", kind, name))
 }
 
 // MarkResourceFailedCreation changes the "Active" condition to false to reflect that a
 // critical resource of the given kind and name was unable to be created.
 func (pas *PodAutoscalerStatus) MarkResourceFailedCreation(kind, name string) {
+	// TODO: This looks more like a bootstrap condition?
```

Comment:
I turned the previous comment into a graph because I wanted to visualize which resources actually update their conditions based on failure to create a child resource (in red): 
![graph](https://user-images.githubusercontent.com/17863526/60216021-d7974780-981d-11e9-8072-84d05e5cd397.png)

It seems like we should be more consistent here... cc @mattmoor